src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2597,7 +2597,7 @@ class C1
 {
     static void Test(int arg1, (byte, byte) arg2)
     {
-        (int, int) t1 = new(int, int)();
+        (int, int)? t1 = new(int, int)?();
```

Comment:
This test does not result in any diagnostics. (only valid cases are here) and there is validation that we do not have diagnostics.

Once some of this cases become errors, the test will catch that.